src/workflows-app/utils/method-common.js

```diff
@@ -96,3 +100,39 @@ export const getFilteredRuns = (filterOption, runsToFilter, errorStates) => {
     return false;
   });
 };
+
+export const getFilteredRunSets = (filterOption, runSetsToFilter, userId, errorStates) => {
```

Comment:
we were using `getFilteredRuns` before, which happened to work because the filterable states are the same across Runs and RunSets, but which inappropriately conflated the abstractions. Now that we want to filter by a runSet-only property (`user_id`), it's time to separate these functions